flutter_inappwebview_web/lib/web/js_bridge.dart

```diff
@@ -46,7 +46,7 @@ external FlutterInAppWebViewBridge? get flutterInAppWebView;
 extension type FlutterInAppWebViewBridge._(JSObject _) implements JSObject {
   external JSObject webViews;
   external JSWebView createFlutterInAppWebView(
-      JSAny viewId, HTMLIFrameElement iframe, HTMLDivElement iframeContainer);
+      String viewId, HTMLIFrameElement iframe, HTMLDivElement iframeContainer);
```

Comment:
You should also update type in `flutter_inappwebview_web/lib/assets/web/web_support.js` and in `nativeCommunication`-related code:
- `flutter_inappwebview_web/lib/web/web_platform.dart`: `JSAny viewId`, `dynamic viewId`
- `flutter_inappwebview_web/lib/web/web_platform_manager.dart`: `webViews` should have `Map<String, dynamic>` type
- `flutter_inappwebview_web/lib/web/in_app_web_view_web_element.dart`: `dynamic _viewId`
- other places where `viewId` is used as `int` or `dynamic`

As is, this PR could break functionality for cases where `viewId` is still used as `int`.

Reply:
thank you @p-mazhnik .. I edited the part of `flutter_inappwebview_web\lib\web\web_platform.dart` to change int to string .. other parts are dynamic which should work without breaking functionality.

Reply:
@ArturLevchuk I have revert it back for now

Reply:
@alikamal1 also as mentioned @p-mazhnik with String value only changed here it brokes callbacks like 'onLoadStart' , 'onLoadStop'... But if we passing value like ```(_viewId as int).toJS``` it works. Anyway callbacks on wasm mode still dead. I tested your solution on version 6.1.5 because on 6.2 callbacks dead on web in general